src/TextEditor.js

```diff
@@ -20,6 +20,16 @@ class TextEditor extends Component {
     this.onChange = this.onChange.bind(this);
     this.handleKeyCommand = this.handleKeyCommand.bind(this);
     this.handleFormating = this.handleFormating.bind(this);
+    this.handleFocus = this.handleFocus.bind(this);
+    this.editorRef = React.createRef();
+  }
+
+  /**
+   * This is a kinda silly hack (but apparently recommended approach) to
+   * making sure the whole visible editor area, not just the first line.
```

Comment:
This comments reads a bit awkward to my 👀. Is it missing a word?

Reply:
Consider changing and then merge?

Reply:
```suggestion
   * making sure the whole visible editor area is clickable, not just the first line.
```